.storybook/components/DesignTokens/Tier1/TypographyTokens.jsx

```diff
@@ -63,25 +63,6 @@ export class Tier1TypographyTokens extends Component {
             })}
           </Grid>
         </Section>
-
-        <Section title="Line Height">
```

Comment:
We don't display any line height tokens since they're legacy

Reply:
Strictly speaking this is not related to the preflight stuff seems like?

Reply:
That is correct, I can revert to scope out if preferred